sunpy/net/helioviewer.py

```diff
@@ -450,3 +435,15 @@ def _request(self, params):
     def _format_date(self, date):
         """Formats a date for Helioviewer API requests"""
         return parse_time(date).isot + "Z"
+
+    def _get_source_id(self, key):
+        """
+        Returns source_id based on the key.
+        """
+        source_id_list = list(partial_key_match(key,self.data_sources))
+        if len(source_id_list) > 1:  # or maybe != 1
+            raise KeyError(f"The values used: {key} do not correspond to one source_id "
+                            "but" f" {len(source_id_list)} source_id(s)." 
```

Comment:
Does this work?

Reply:
Yeah though it is a bit weird. I will send a screenshot
![error](https://user-images.githubusercontent.com/31286078/53353423-a9c2f480-394a-11e9-8161-d829c86c1522.png)